### PR TITLE
blog post - homelab networking video writeup

### PR DESCRIPTION
**Pull request by ironicbadger**
blog post - homelab networking video writeup

### CODE REVIEW

File: data/blog/homelab-networking-vlans.mdx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+summary: 'Create a segmented network zone with Proxmox, OPNsense, and Tailscale in a new how-to video.'
+tags:
+  - Homelab
+  - Networking
+images: ['/images/homelab-networking-vlans/social.png']
+---
+
+It’s OK to break stuff, that’s how we learn. But all too often it’s impossible to find a safe space to do so. In this video, Alex breaks down how to create a segmented network zone for learning purposes using Proxmox to virtualize OPNsense.
+
+Imagine you'd like to test out a new way of working with DHCP — the technology responsible for managing IP addresses within a network. A service like this is absolutely critical to the basic functionality of your network. This often means your window to tinker with different options is often limited to minutes. When you take something as core to a networks function, like DHCP, and break it, it means everything from your laptop to your [robot vacuum](https://tailscale.dev/blog/tailscale-sucks) breaks too.
```
Comment (WalterHub) [inline data/blog/homelab-networking-vlans.mdx:14]:
```suggestion
Imagine you'd like to test out a new way of working with DHCP — the technology responsible for managing IP addresses within a network. A service like this is absolutely critical to the basic functionality of your network. This often means your window to tinker with different options is often limited to minutes. When you take something as core to a network's function, like DHCP, and break it, it means everything from your laptop to your [robot vacuum](https://tailscale.dev/blog/tailscale-sucks) breaks too.
```

File: public/images/homelab-networking-vlans/social.png
Comment (WalterHub) [inline public/images/homelab-networking-vlans/social.png:1]: "Opnsense" in social.png should be "OPNsense"

**Review by WalterHub — COMMENTED** [review]
LGTM, added a suggestion and a comment

**Review by thisisparker — APPROVED** [review]
Seconding, LGTM!

File: public/images/homelab-networking-vlans/social.png
Comment (ironicbadger) [reply public/images/homelab-networking-vlans/social.png:1]: well spotted. i would need to reup the youtube thumbnail as well. i will submit a separate PR to address this one later.

**Pull request merged**